src/app/common/stores/applications/Distribution/Wallet.store.ts

```diff
@@ -1,14 +1,28 @@
 import Web3 from 'web3';
 import Notify from 'bnc-notify';
-import Onboard from 'bnc-onboard';
+import Onboard from '@web3-onboard/core';
 import { Contract } from 'web3-eth-contract';
+import injectedModule from '@web3-onboard/injected-wallets';
 import { action, computed, makeObservable, observable } from 'mobx';
 import config from '~app/common/config';
+import { getImage } from '~lib/utils/filePath';
 import BaseStore from '~app/common/stores/BaseStore';
 import Wallet from '~app/common/stores/Abstracts/Wallet';
-import { wallets } from '~app/common/stores/utilis/wallets';
 import NotificationsStore from '~app/common/stores/applications/SsvWeb/Notifications.store';
 import DistributionStore from '~app/common/stores/applications/Distribution/Distribution.store';
+import {
+  changeCurrentNetwork, getCurrentNetwork,
+  inNetworks,
+  NETWORKS, notIncludeMainnet, testNets,
+} from '~lib/utils/envHelper';
+
+const WALLET_CONNECTED = 'WalletConnected';
+
+const TOKEN_NAMES = {
```

Comment:
Can these be defined in a configuration rather then in the code itself?

Reply:
done